codegenerator/cli/templates/static/codegen/src/db/TrackTables.res

```diff
@@ -227,10 +233,12 @@ let trackAllTables = async () => {
   Logging.info("Tracking tables in Hasura")
 
   let _ = await clearHasuraMetadata()
-  await [Db.allStaticTables, Db.allEntityTables]
-  ->Belt.Array.concatMany
-  ->Utils.Array.awaitEach(async ({tableName}) => {
-    await trackTable(~tableName)
+  let tableNames =
+    [Db.allStaticTables, Db.allEntityTables]
+    ->Belt.Array.concatMany
+    ->Js.Array2.map(({tableName}) => tableName)
+  await trackTables(~tableNames)
```

Comment:
I guess once the tables are all tracked the relationships and permisions can also be setup concurrently as well?